jbmc/regression/jbmc-strings/string-non-empty-option/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+Test.class
+--function Test.checkMinLength
+^EXIT=10$
+^SIGNAL=0$
+assertion.* line 11 function java::Test.checkMinLength.*: FAILURE
+assertion.* line 13 function java::Test.checkMinLength.*: FAILURE
+assertion.* line 17 function java::Test.checkMinLength.*: FAILURE
+assertion.* line 19 function java::Test.checkMinLength.*: FAILURE
+
```

Comment:
No blank lines here please, it confuses test runs on Windows. The same applies to the other *.desc files in this commit.